index.md

```diff
@@ -9,21 +9,22 @@ The Standard for Public Code gives cities a model for building their own open so
 * [Introduction and background](introduction.md)
 * [Readers guide: how to interpret this standard](readers-guide.md)
 * [Glossary](glossary.md)
-* [Criteria](criteria/)
-  * [Code in the open](criteria/code-in-the-open.md)
-  * [Bundle policy and source code](criteria/bundle-policy-and-code.md)
-  * [Create reusable and portable code](criteria/reusable-and-portable-codebases.md)
-  * [Welcome contributions](criteria/open-to-contributions.md)
-  * [Maintain version control](criteria/version-control-and-history.md)
-  * [Require review of contributions](criteria/require-review.md)
-  * [Document your objectives](criteria/document-objectives.md)
-  * [Document your code](criteria/documenting.md)
-  * [Use plain English](criteria/understandable-english-first.md)
-  * [Use open standards](criteria/open-standards.md)
-  * [Use continuous delivery](criteria/continuous-delivery.md)
-  * [Publish with an open license](criteria/open-licenses.md)
-  * [Use a coherent style](criteria/style.md)
-  * [Pay attention to codebase maturity](criteria/advertise-maturity.md)
+* [Community criteria](community-criteria/)
+  * [Code in the open](community-criteria/code-in-the-open.md)
+  * [Welcome contributions](community-criteria/open-to-contributions.md)
+* [Codebase criteria](codebase-criteria/)
+  * [Bundle policy and source code](codebase-criteria/bundle-policy-and-code.md)
+  * [Create reusable and portable code](codebase-criteria/reusable-and-portable-codebases.md)
+  * [Maintain version control](codebase-criteria/version-control-and-history.md)
+  * [Require review of contributions](codebase-criteria/require-review.md)
```

Comment:
A question: Is this perhaps also a community criterion?

Reply:
I suggest that "require review" is a MUST, but SHOULD choose reviewers which are cross-organization or cross-context, thus the MUST is more codebase and the SHOULD is more community. Since the MUST is codebase, I suggest it stays as codebase for now.